src/app/components/school-menu/school-list.tsx

```diff
@@ -12,10 +12,15 @@ interface SchoolListProps {
   color: 'dark' | 'light';
 }
 
-export const SchoolList = ({ list, color }: SchoolListProps) => (
-  <ul className="school-list">
-    {list?.map((item, index) => (
-      <SchoolItem item={item} index={index} key={index} color={color} />
-    ))}
-  </ul>
-);
+export const SchoolList = ({ list, color }: SchoolListProps) => {
+  const className = (!!list && !!list[0] && 'description' in list[0]) 
+    ? "school-list"
+    : "school-list school-list_width";
+  return (
+    <ul className={className}>
+      {list?.map((item, index) => (
+        <SchoolItem item={item} index={index} key={index} color={color} />
```

Comment:
Index for a key? Do we have other unique options?

Reply:
changed to title